lib/src/platform/src/platform.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+import '../../error/error.dart';
+import '../../generated/platform_constants.dart';
+import '../platform.dart';
+
+class Platform {
```

Comment:
I think it makes the code easier to read when using it/ importing, because importing the original file without a class will pollute the namespace with all the methods from the imported file. Now, they are all encapsulated in a `Platform` class. It is also easier to import, since Android Studio will not suggest importing a file `as Platform` (the class writer would know to `import ... as Platform`, but future users would just import directly using autoimport. 

TLDR: I was trying to remove the (arguably) _global_ methods, which is what the file originally had when writing code.

Reply:
Alright.